main/src/ca/uwaterloo/flix/language/phase/typer/PrimitiveEffects.Methods.json

```diff
@@ -1,61 +1,5 @@
 {
   "methods": {
-    "java.lang.Boolean::getBoolean": "Env, IO",
-    "java.lang.Byte::describeConstable": "Sys, IO",
-    "java.lang.Double::resolveConstantDesc": "Sys, IO",
-    "java.lang.Float::resolveConstantDesc": "Sys, IO",
-    "java.lang.Integer::describeConstable": "Sys, IO",
-    "java.lang.Integer::getInteger": "Env, IO",
-    "java.lang.Integer::resolveConstantDesc": "Sys, IO",
-    "java.lang.Long::describeConstable": "Sys, IO",
-    "java.lang.Long::getLong": "Env, IO",
-    "java.lang.Long::resolveConstantDesc": "Sys, IO",
-    "java.lang.Short::describeConstable": "Sys, IO",
-
-    "java.lang.ClassLoader::getDefinedPackage": "IO",
-    "java.lang.ClassLoader::getDefinedPackages": "IO",
-    "java.lang.ClassLoader::getName": "IO",
-    "java.lang.ClassLoader::getPackages": "IO",
-    "java.lang.ClassLoader::getParent": "IO",
-    "java.lang.ClassLoader::getPlatformClassloader": "Sys, IO",
-    "java.lang.ClassLoader::getResource": "FsRead, IO",
-    "java.lang.ClassLoader::getResourceAsStream": "FsRead, IO",
-    "java.lang.ClassLoader::getResources": "FsRead, IO",
-    "java.lang.ClassLoader::getSystemClassLoader": "Env, IO",
-    "java.lang.ClassLoader::getSystemResource": "Env, FsRead, IO",
-    "java.lang.ClassLoader::getSystemResourceAsStream": "Env, FsRead, IO",
-    "java.lang.ClassLoader::getSystemResources": "Env, FsRead, IO",
-    "java.lang.ClassLoader::getUnnamedModule": "IO",
-    "java.lang.ClassLoader::isRegisteredAsParallelCapable": "IO",
-    "java.lang.ClassLoader::loadClass": "Sys, IO",
-    "java.lang.ClassLoader::resources": "FsRead, IO",
-    "java.lang.ClassLoader::setClassAssertionStatus": "Sys, IO",
-    "java.lang.ClassLoader::setDefaultAssertionStatus": "Sys, IO",
-    "java.lang.ClassLoader::setPackageAssertionStatus": "Sys, IO",
-
-    "java.lang.Runtime::addShutdownHook": "Exec, Sys, IO",
-    "java.lang.Runtime::availableProcessors": "Env, IO",
-    "java.lang.Runtime::exec": "Env, Exec, IO",
-    "java.lang.Runtime::exit": "Sys, IO",
-    "java.lang.Runtime::freeMemory": "Env, NonDet, IO",
```

Comment:
A NonDet was removed here. Wouldn't it make sense for this PR not to touch NonDet? it seems a bit muddled on top of the removal stuff

Reply:
What do you mean exactly? I think the only muddle is that I left non-det for java.util.Random.

Otherwise I removed everything except purity annotations for the wrapper types.

Reply:
On master `java.lang.Runtime::freeMemory` produces `NonDet`, with this change it doesn't.

Can't this PR just avoid touching NonDet if we are not removing it?

Reply:
This PR removes all annotations of Java except for wrappers (and a few misc things)